src/client/assets/styles/editor.scss

```diff
@@ -13,6 +14,36 @@
         }
     }
 
+    view-mode {
+        .excel {
+            @include theme($excel);
+        }
+
+        .word {
+            @include theme($word);
+        }
+
+        .powerpoint {
+            @include theme($powerpoint);
+        }
+
+        .onenote {
+            @include theme($onenote);
+        }
+
+        .project {
+            @include theme($project);
+        }
+
+        .teams {
+            @include theme($teams);
+        }
+
+        .web {
+            @include theme($web);
+        }
```

Comment:
Can discuss with you more offline about the theme logic; the reason that we even need this code is that the existing `applyTheme()` function from `theme.ts` explicitly loads the CSS stylesheets through the `NodeRequire` module. As a result, when you try to load the same stylesheet twice (say you load a gist for excel then word and then back to excel) it'll keep the previous one and not re-load it (so the gist for excel would be loaded but with a word theme instead of the excel theme).